packages/shared/src/components/post/collection/CollectionPostHeaderActions.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { ReactElement, useContext } from 'react';
+import classNames from 'classnames';
+import classed from '../../../lib/classed';
+import { SimpleTooltip } from '../../tooltips/SimpleTooltip';
+import { Button } from '../../buttons/Button';
+import SettingsContext from '../../../contexts/SettingsContext';
+import { Origin } from '../../../lib/analytics';
+import BellIcon from '../../icons/Bell';
+import { PostHeaderActionsProps } from '../common';
+import { PostMenuOptions } from '../PostMenuOptions';
+
+const Container = classed('div', 'flex flex-row items-center');
+
+const CollectionPostHeaderActions = ({
+  onShare,
+  post,
+  onClose,
+  inlineActions,
+  className,
+  notificationClassName,
+  contextMenuId,
+  onRemovePost,
+  ...props
+}: PostHeaderActionsProps): ReactElement => {
+  const { openNewTab } = useContext(SettingsContext);
+
+  return (
+    <Container {...props} className={classNames('gap-2', className)}>
+      <SimpleTooltip
+        placement="bottom"
+        content="Subscribe"
+        disabled={!inlineActions}
+      >
+        {/* TODO: FIX ACTION ON THIS with useNotificationPreference */}
```

Comment:
This needs to be addressed in this PR

Reply:
We are gonna do it separate since some other work is requred, added ticket.